src/mlpack/methods/ann/layer/atrous_convolution.hpp

```diff
@@ -257,6 +257,15 @@ class AtrousConvolution
   //! Modify the internal Padding layer.
   ann::Padding<>& Padding() { return padding; }
 
+  //! Get size of the weight matrix.
+  size_t WeightSize() const
+  {
+    size_t weightSize = (outSize * inSize * kernelWidth * kernelHeight) +
+        outSize;
+    return weightSize;
```

Comment:
```suggestion
    return (outSize * inSize * kernelWidth * kernelHeight) +
        outSize;
```

Just to keep it short.